repairnator/repairnator-realtime/src/main/java/fr/inria/spirals/repairnator/realtime/RTScanner.java

```diff
@@ -40,25 +42,24 @@
     private String runId;
     private List<SerializerEngine> engines;
     private BlacklistedSerializer blacklistedSerializer;
-    private Duration duration;
     private EndProcessNotifier endProcessNotifier;
 
     public RTScanner(String runId, List<SerializerEngine> engines) {
+        this.config = RepairnatorConfig.getInstance();
         this.engines = engines;
         this.blackListedRepository = new ArrayList<>();
         this.whiteListedRepository = new ArrayList<>();
         this.tempBlackList = new HashMap<>();
         this.buildRunner = new BuildRunner(this);
         this.inspectBuilds = new InspectBuilds(this);
+        this.inspectBuilds.setMaxSubmittedBuilds(this.config.getMaxInspectedBuilds());
```

Comment:
Actually if the value are available in the config, those setter are useless too: we can do the set directly inside the constructor of the class no?

Reply:
Oh, yes!